### PR TITLE
Speedtesttracker integration

### PR DESCRIPTION
**Pull request by boerniee**
## Description

Added a service for the [SpeedtestTracker project](https://github.com/henrywhitaker3/Speedtest-Tracker). 

Fixes # (issue)

## Type of change

- [  ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [  ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] I've read & comply with the [contributing guidelines](https://github.com/bastienwirtz/homer/blob/main/CONTRIBUTING.md)
- [X] I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers. 
- [X] I have made corresponding changes the documentation (README.md).
- [X] I've checked my modifications for any breaking changes, especially in the `config.yml` file

### CODE REVIEW

**Comment by PedroC1999** [comment]
Hello!~

What additional work is required for this to be merged? interested in getting this closer to the PR being accepted :)

File: src/components/services/SpeedtestTracker.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<template>
```
Comment (Roundaround) [inline src/components/services/SpeedtestTracker.vue:1]: Please extend the Generic service and modify only the #content / subtitle portion

**Review by Roundaround — CHANGES_REQUESTED** [review]

**Comment by bastienwirtz** [comment]
Hey there, 
Sorry for letting you wait so long ! I'm will merge this in a new branch to update it before merging it. 
Thank for your contribution !

**Comment by bastienwirtz** [comment]
There you go: https://github.com/bastienwirtz/homer/blob/main/src/components/services/SpeedtestTracker.vue
This is merged on main :+1:

**Pull request merged**